modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -388,6 +388,10 @@ function getComments($issueID)
 
     //looping by reference so can edit in place
     foreach ($unformattedComments as &$comment) {
+        $comment["dateAddedFormatted"] = date(
```

Comment:
I'm not familiar with what date formatting utilities JavaScript has. I did want to do it only on the front-end. MVC and separation of responsibilities and whatnot. But I didn't know so I shrugged and went with this approach.